actions/status-checker/action.yml

```diff
@@ -3,8 +3,14 @@ description: "Checks the OpenPublishing.Build's status check's description to se
 author: 'Genevieve Warren'
 inputs:
   repo-token:
-    description: 'The GITHUB_TOKEN secret'
+    description: 'The GITHUB_TOKEN secret. Requires the following permissions: statuses: write, issues: write, pull-requests: write.'
     required: true
+  docs-path:
+    description: 'The path to the docs folder, such as; "docs" for the .NET docs repo.'
+    default: 'docs'
+  vanity-slug:
```

Comment:
Can we call this URL base path instead? It took me a while to figure out what "vanity slug" meant.

Reply:
Yeah, that's fine. I'll update that.